app/src/main/java/com/portfolio/tasky/views/TaskyAppCompatEditText.kt

```diff
@@ -0,0 +1,62 @@
+package com.portfolio.tasky.views
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.res.TypedArray
+import android.graphics.drawable.Drawable
+import android.text.InputType
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.FrameLayout
+import androidx.appcompat.widget.AppCompatEditText
+import com.portfolio.tasky.R
+import com.portfolio.tasky.databinding.LayoutInputFieldBinding
+
+class TaskyAppCompatEditText(context: Context, attrs: AttributeSet) : FrameLayout(context, attrs)   {
+
+    private lateinit var viewBinding: LayoutInputFieldBinding
+    private var error : Boolean = false
+    private var drawableLast : Drawable? = null
+    private lateinit var editText : AppCompatEditText
+    private var inputType : Int = 0
+    private var hint : String? = null
+
+    init {
+        setupView(attrs)
+        isError()
+        setDrawableLast()
+        setInputType()
+        setHint()
+    }
+
+    private fun setHint() {
+        viewBinding.hint = hint
+    }
+
+    private fun setInputType() {
+        viewBinding.etInput.inputType = inputType
+    }
+
+    private fun setDrawableLast() {
+        viewBinding.drawableLast = drawableLast
+    }
+
+    fun isError(){
+        viewBinding.error = error
+    }
+
+    @SuppressLint("CustomViewStyleable")
+    private fun setupView(attrs: AttributeSet) {
+        val inflater = context.getSystemService(Context.LAYOUT_INFLATER_SERVICE) as LayoutInflater
+        viewBinding = LayoutInputFieldBinding.inflate(inflater, this, false)
+        addView(viewBinding.root)
+
+        editText = viewBinding.etInput
+        val ta: TypedArray = context.obtainStyledAttributes(attrs, R.styleable.EditTextInputField)
```

Comment:
Make variable names more readable

``` val typedArray ``` is way easier to read than ``` val ta ```

Reply:
Yes!

Done